doc/api/child_process.md

```diff
@@ -325,6 +325,8 @@ changes:
 * `args` {string[]} List of string arguments.
 * `options` {Object}
   * `cwd` {string} Current working directory of the child process.
+  * `detached` {boolean} If `true`, prepares child to run
```

Comment:
Probably better to duplicate the text for the `detached` option under `spawn()` rather than have two different phrasings like this?